src/index.ts

```diff
@@ -15,8 +15,14 @@ export default function(): void {
   usePlugin('@nomiclabs/buidler-etherscan')
 
   // Task definitions.
-  require(path.join(__dirname, '/tasks/start-task'))
-  require(path.join(__dirname, '/tasks/publish'))
+  // Note: Tasks must be setup in a function. If task() is run in the
+  // module body on test teardown the they will not be setup again
```

Comment:
```suggestion
  // module body on test teardown, they will not be setup again
```

But not sure I really get what's going on with the tests even with this comment.

Reply:
I'll try to phrase it better!
> buidler sets up the environment by appending tasks to it. This happens when a `task()` function is called. Therefore, `task()` must be called on **every** test run, not only on the first time. However, when node imports a file with `require` the module body will only be run once on the first time and then use the cached result.

Does that work better?

Reply:
Ah yes, this makes a lot more sense now!